docs/user/manifests/kubeapps-local-dev-users-rbac.yaml

```diff
@@ -0,0 +1,34 @@
+# kubeapps-operator has cluster-admin
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: kubeapps-operator
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: User
+  name: oidc:kubeapps-operator@example.com
```

Comment:
I guess you need to set the prefix `oidc:` here because you have configured the API with that prefix. Is that intentional to be more explicit? I have it without prefix in my dev cluster, just in case you prefer to do that without the prefix.

Reply:
Yeah, it's intentional (and was recommended) so it's clear in k8s where the authn is coming from (when assigning RBAC etc.)